### DISCUSSION TITLE
VSCode - GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

### DISCUSSION THREAD

**Discussion by marc-aurele-besner** [community/community#19361] [Copilot Conversations]
Like everyone here, I have been using copilot since the public beta, I selected a plan this week and fill in my payment info. It started working again on my work pc, but my personal pc now has this error msg "GitHub Copilot could not connect to server. Extension activation failed: "User not authorized""
re-Installing the extension, going true the web activation again, ... does not fix the issue...
(both pc use my personal github account)

Anyone has a clue?

**Comment by riversdiniz** [comment]
Guys, good morning!

When using and installing the copilot github extension is it necessary to do some configuration in vscode?

**Reply by marc-aurele-besner** [reply]
There is not much config to be done in VScode, else than if the extension is conflicting with other extension (especially for the use of some shortcut) or to change the default shortcut. (to my understanding)

**Reply by riversdiniz** [reply]
tks!

**Comment by 13kparkin** [comment]
I am having this same issue. I just got invited to use it via email and when I go to install it I get the error "GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"".

**Comment by gpant** [comment]
The same issue exactly, I already had payment info (github pro sub), I have enabled copilot and I get the same error message.

**Comment by manuarora700** [comment]
Same issue

**Comment by JaimeTR** [comment]
I installed GitHub Copilot Nightly, and it fixed the problem for me.
Before installing GitHub Copilot Nightly, please uninstall or disable Github copilot and install the extension from https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly

If this answer is helpful, please 'Mark as answer'. :)

**Reply by marc-aurele-besner** [reply]
This solved the issue thank you!

**Comment by gpant** [comment]
I had to contact github, they changed something in the account and then when I logged in, it was fixed.

**Reply by marc-aurele-besner** [reply]
Thank you, I just contacted them, hopefully they will also solve the issue for me 😄
